Tools/CISupport/ews-build/config.json

```diff
@@ -199,6 +199,11 @@
       "factory": "StyleFactory", "platform": "*",
       "workernames": ["ews151", "ews253", "webkit-misc"]
     },
+    {
+      "name": "Validate-User-EWS", "shortname": "validate-user",
+      "factory": "ValidateUserFactory", "platform": "*",
+      "workernames": ["webkit-misc"]
```

Comment:
(I left this value in because I'm still unclear on how or where this is meant to be deployed.)